client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -13,73 +13,102 @@
-          <li>
-            {{ rowData.institution }}
-          </li>
-          <li class="o-list__item o-hellip--nowrap">
-            {{ date(rowData.createdDate) }}
-          </li>
-        </ul>
-      </template>
-      <template v-slot:tags="rowData">
-        <div v-if="!rowData.edit">
-          <span>
-            {{ separateByCommas(rowData.tags) }}
-          </span>
-        </div>
-        <dp-multiselect
-          v-else
-          v-model="editingInstitutionTags"
-          :options="tagList"
-          label="name"
-          track-by="id"
-          multiple />
-      </template>
-      <template v-slot:action="rowData">
-        <div class="float-right">
-          <template v-if="!rowData.edit">
-            <button
-              :aria-label="Translator.trans('item.edit')"
-              class="btn--blank o-link--default"
-              @click="editInstitution(rowData.id)">
-              <i
-                class="fa fa-pencil"
-                aria-hidden="true" />
-            </button>
-          </template>
-          <template v-else>
-            <button
-              :aria-label="Translator.trans('save')"
-              class="btn--blank o-link--default u-mr-0_25"
-              @click="addTagsToInstitution(rowData.id)">
-              <dp-icon
-                icon="check"
-                aria-hidden="true" />
-            </button>
-            <button
-              class="btn--blank o-link--default"
-              :aria-label="Translator.trans('abort')"
-              @click="abortEdit()">
-              <dp-icon
-                icon="xmark"
-                aria-hidden="true" />
-            </button>
-          </template>
-        </div>
-      </template>
-    </dp-data-table>
+
+    <dp-loading
+      class="u-mt"
+      v-if="isLoading" />
+    <template v-else>
+      <dp-column-selector
+        data-cy="institutionList:selectableColumns"
+        :initial-selection="currentSelection"
+        local-storage-key="institutionList"
+        :selectable-columns="selectableColumns"
+        use-local-storage
+        @selection-changed="setCurrentSelection" />
+
+      <dp-data-table
+        ref="dataTable"
+        class="u-mt-2 overflow-x-auto scrollbar-none"
+        data-dp-validate="tagsTable"
+        data-cy="institutionList:dataTable"
+        :header-fields="headerFields"
+        is-resizable
+        :items="institutionList"
+        track-by="id">
+        <template v-slot:name="institution">
+          <ul class="o-list max-w-12">
+            <li>
+              {{ institution.name }}
+            </li>
+            <li class="o-list__item o-hellip--nowrap">
+              {{ date(institution.createdDate) }}
+            </li>
+          </ul>
+        </template>
+        <template
+          v-for="category in institutionTagCategories"
+          v-slot:[category.attributes.name]="institution">
+          <div v-if="!institution.edit">
+            <span>
```

Comment:
why do You need the span?

Reply:
We don't, I removed it with [9845678](https://github.com/demos-europe/demosplan-core/pull/4115/commits/9845678e475ae47fe8ca5fa4e8bd7822d3505762)